src/openeo_test_suite/tests/workflows/test_load_apply_save.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import Path
+import xarray as xr
+
+def test_load_apply_save_netcdf(connection,bounding_box_small,temporal_interval_one_day,s2_stac_url):
+    from openeo.processes import clip
+    filename = "test_load_apply_save_netcdf.nc"
+    Path(filename).unlink(missing_ok=True)
```

Comment:
Instead of this DIY-cleanup stuff, it's recommended to use the `tmp_path` built-in fixture: https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html :
- automatically creates a temp folder to work in
- avoids polluting your repo with temp output you might accidentally commit
-  clean up (removing the temp folder) is automatic, so you don't have to try doing that yourself (which is very tedious to do right)
- `tmp_path` also comes with history features, so that you can inspect results from the last 3 runs e.g.